xnat_tools/xnat2bids.py

```diff
@@ -92,6 +94,22 @@ def xnat2bids(
         overwrite=overwrite,
         cleanup=cleanup,
     )
+
+    bids_experiment_dir = glob.glob(f"{bids_root_dir}/*/*/bids")[0]
+
+    bids_postprocess(
+        bids_experiment_dir,
+        user=user,
+        password=password,
+        session="",
```

Comment:
hm... shouldn't the `session` here inherit from the input to this executable? If we run xnat2bids on only one XNAT_SESSION, why is the post processing running on all?

Same applies to the overwrite and logfile

Reply:
I'm not sure why this one god resolved... I don't think I see the changes addressed or maybe it should be as is and I'm not following?

Reply:
@mirestrepo I am passing the subject and session_suffix returned from dicom_export, rather than the Accession ID passed in session.  This eliminates the redundant request to XNAT for subject and session_suffix information.